vault/vault/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# base image
```

Comment:
Vault dockerfile has to be stored into a different repository.
Like that it can be better if someone wanna to use terraform or something else....